palm/plugins/multi_service/commands/cmd_down.py

```diff
@@ -0,0 +1,14 @@
+import click
+
+
+@click.command("down")
+@click.argument("flags", nargs=-1)
+@click.pass_obj
+def cli(environment, flags: tuple):
+    """Bring up all services"""
+    # This sucks. Don't commit this.
+    flags = [f"-{flag}" if len(flag) == 1 else f"--{flag}" for flag in flags]
```

Comment:
can you just do `[f"--{flag}" for flag in flags]` and let it join the single element?